utils/validation.js

```diff
@@ -109,6 +120,31 @@ class Validation {
 
         return true;
     }
+
+    /**
+     * Tests the given value against the rules of dimensions (depth/width).
+     *
+     * Dimensions cannot be negative.
+     *
+     * @since 1.0.0
+     *
+     * @param {number}  value   A value to test against the span rules.
+     * @param {string}  field   A string representation of the field name for
+     *                          the error message.
+     * @param {string}  caller  A string representation of the caller's class
+     *                          for the error message.
+     *
+     * @returns {boolean}       true if the dimension is valid, false
+     *                          otherwise.
+     */
+    static testDimensionValue(value, field, caller) {
+        if (value < 0) {
+            console.log(caller + ': Field "' + field + '" cannot be negative.');
```

Comment:
It dawns on me that at some point were going to have to change how the logs are emitted so we can catch them and send them to the user through the web page rather than the console.

Reply:
Right, I've been doing this for now because I'm not quite sure how the actual displaying text on the page works. I plan on having some object or string that I pass down through the validate functions to create a sort of stack trace. That way I can organize and actually name which objects are invalid.